modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -583,6 +585,29 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private List<ObjectRelationship> _getManyToManyObjectRelationships(
+			long objectDefinitionId)
+		throws PortalException {
+
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipLocalService.
+				getObjectRelationshipsByObjectDefinitionId2(objectDefinitionId);
+
+		return TransformUtil.transform(
+			objectRelationships,
+			objectRelationship -> {
+				if (StringUtil.equals(
+						objectRelationship.getType(),
+						ObjectRelationshipConstants.TYPE_MANY_TO_MANY) &&
+					!objectRelationship.isReverse()) {
+
+					return objectRelationship;
+				}
+
+				return null;
+			});
+	}
+
```

Comment:
@danvb112 @marcelabc I think this method should be inside ObjectRelationshioLocalService. That way, we avoid processing over a list.